python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -964,7 +962,7 @@ def plotMatches(self, matches, filterName, filenamer, dataId, description="match
                                    cat["distance"]*(1.0*afwGeom.radians).asArcseconds()*self.unitScale,
                                "Distance (%s) (calib_astrom_used)" % unitStr, shortName,
                                self.config.analysisMatches, prefix="src_", goodKeys=["calib_astrometry_used"],
-                               qMin=-0.01*matchRadius, qMax=0.5*matchRadius,
+                               qMin=-0.01*self.matchRadius, qMax=0.5*self.matchRadius,
```

Comment:
I don't follow why qMin and qMax are based on `self.matchRadius` instead of `matchRadius`

Reply:
Ah, that was part of my removal of the erroneous matchRadius being plotted to the reference-matched plots (the matching was actually done in the stack in this case.  Here, we are just reading in those matches.)  I accomplish this by allowing for a `matchRadius` to be `None`, but still wanted a good guess at a good scaling for the y-axis and the "best guess" I have is the default for when we do our own matching (this does assume the two cases will have had roughly the same tolerance...but some dynamic tweaking will happen if the limits are not satisfactory according to the list of "criteria" in the `Analysis` class).

I do agree that this should be set to `matchRadius` if one is passed in, so I now do the scaling by a new variable defined as:
`qMatchScale = matchRadius if matchRadius else self.matchRadius`